### ISSUE TITLE
Using vue-carousel with AEM

### ISSUE THREAD

**Issue by ghost** [SSENSE/vue-carousel#106]
I have built my own custom vue component that uses `vue-carousel` within itself.
It has all been great until we put the component into AEM... now on Preview mode - we are getting an error that says `SyntaxError: expected property name, got ')'`. This error blocks ALL functionality on the page & blocks the developer from being able to go back to the Edit page - it's not until the user goes to another (NON CAROUSEL) page and sets the view back to Preview that we can then go back to our carousel page and be in edit view...

```
// AEM CONTAINER TMPL
<my-carousel>
    <sly data-sly-test="${wcmmode.edit}">
        <div>
    </sly>

    <sly data-sly-resource="${'caruselpar' @ resourceType='foundation/component/parsys'}"></sly>

    <sly data-sly-test="${wcmmode.edit}">
        </div>
    </sly>
</my-carousel>
```

```
// VUE COMPONENT TMPL
<template>
  <div class="carousel">
    <carousel :perPage="1"
      :paginationEnabled="true"
      :navigationEnabled="true"
      :navigationNextLabel="getChevron(rhCta)"
      :navigationPrevLabel="getChevron(lhCta)"
      paginationActiveColor="#000"
      paginationColor="none"
      :paginationPadding="20"
      :paginationSize="20"
      :loop="true"
      :minSwipeDistance="80">

      <slide v-for="slide in slides" :key="slide" v-html="slide"></slide>
    </carousel>
  </div>
</template>
```

**Comment by quinnlangille** [comment]
Hey @Francis-Leigh, sorry for the late reply. Have you managed to get this to work since? It seems like it's more of a build issue inside AEM to me. If you did get it to work, I'd be interested to hear if the fix was on the carousel or AEM end?

**Comment by quinnlangille** [comment]
Haven't heard back here so going to close this, feel free to open another issue if there are still any problems :~)
